wkcuber/__main__.py

```diff
@@ -78,7 +75,7 @@ def create_parser():
         )
 
     else:
-        downsample_mags(
+        downsample_mags_isotropic(
```

Comment:
I think the default should be anisotropic downsampling.

Reply:
Yes I think so too but this may change the result of the other code since now all data sets that were scaled isotropic before will now be scaled anisotropic.

Reply:
This `__main__.py` is only run from the command line not from other code.